### ISSUE TITLE
Suggestion: Changing the scale in statistics display

### ISSUE THREAD

**Issue by MarkMcWire** [widelands/widelands#6217]
It would be desirable if the scale of the Y-axis aligned with the currently displayed maximum and not with the highest value in the game.

If you zoom in on the time scale, the lines are often quite far down because the highest value in the game is still used for the display. Even if the current values are much lower. However, the highest value in the selected time interval should be used to adjust the Y-axis.

Example: If the highest military strength in the game is 4000, but the maximum in the last 10 hours is 2500, 2500 should be used to display the diagram for the last 10 hours interval.

The background to my suggestion is that it is currently difficult to recognize changes or trends in the diagram when you zoom in the time scale because the values are scaled based on the global maximum value and not at the local maximum value of the selected section.
